frontend/src/views/home/index.tsx

```diff
@@ -4,23 +4,39 @@ import { Link } from 'react-router-dom';
 import BookList from '../../components/home/bookList';
 import Spinner from '../../components/spinner';
 import { stockData } from '../../data';
-import { ENDPOINTS } from '../../utils/endpoint';
-import { useBook } from './hook/useBook';
+import { useBooks } from './hook/useBooks';
 
+/**
+ * Home component is used to show the list of books.
+ *
+ * @returns {React.ReactElement} A react component that renders the list of books.
+ *
+ * @example
+ * <Home />
+ *
+ */
 function Home() {
```

Comment:
Regression found.
Try to find it as an exercise.

Hint: set a LIMIT that will shows up pagination, then change page. What happens?